monitoring/prometheus/alerts/node-health.yml

```
@@ -0,0 +1,68 @@
+  - name: demos-node-health
+    rules:
+      - alert: NodeDown
+        # Catches both scrape failure (up == 0) AND target removed
+        # from service discovery (no up series at all).
+        expr: up{job="demos-node"} == 0 or absent(up{job="demos-node"})
+        for: 1m
+        labels:
+          severity: critical
+        annotations:
+          summary: Demos node is down
+          description: "Prometheus failed to scrape `demos-node` for >1m, OR the target was removed from service discovery. Node may have crashed or metrics server is unreachable."
```

YAMLlint (1.38.0)

[error] string value is redundantly quoted with any quotes

(quoted-strings)